.release.json

```diff
@@ -30,8 +30,14 @@
         "git commit -nm \"Preparing gh-pages\"",
         "git subtree add --prefix dist/docs origin gh-pages",
         "npm run build",
-        "cp -r examples dist/docs/examples",
-        "cp -r dist/build/* dist/docs/examples/virtualbackground/",
+        "MAJOR_VERSION=$(echo ${RELEASE_VERSION} | cut -d'.' -f1)",
+        "mkdir -p dist/temp/examples",
+        "cp -r examples/* dist/temp/examples/",
+        "cp -r dist/build/* dist/temp/examples/virtualbackground/",
+        "mv dist/docs/* dist/temp/",
+        "mkdir -p dist/docs/v${MAJOR_VERSION}.x",
+        "mv dist/temp/* dist/docs/v${MAJOR_VERSION}.x/",
+        "rm -rf dist/temp",
```

Comment:
I could have changed the documentation output configuration dynamically instead of creating this temp folder, but limiting the change to this file where I am certain regarding the `RELEASE_VERSION` value was clearer.